posts/enterprise-lacking-cloud-skills.md

```diff
@@ -0,0 +1,63 @@
+# Enterprise lacking skills required for cloud migration
+
+> 66% of I.T. leaders say their organisations are not adequately prepared for the disruptions to come
+
+http://www.gartner.com/newsroom/id/3375817
+
+For all the excitement and talk of cloud technologies it seems the reality is a little less glamorous. With more and more executives backing the move to the cloud, more and more organisations are booting up large migration programmes. The problem is, organisations do not necessarily have the skills to migrate. 66% of I.T. leaders agree in a Gartner report that their organisations are not adequately prepared for this change.
+
+> 81% of enterprises rely on service providers for cloud
+
+https://www.veritas.com/news-releases/2016-06-28-state-of-the-hybrid-cloud-research
+
+Not only that but enterprises are reporting the need to outsource this to service providers, often locking themselves into projects lasting years and ultimately huge bills. 81% of enterprises according to a Veritas survey are relying on service providers to help them migrate.
+
+> 35% have migrated applications and infrastructure to the cloud that were ultimately brought back on-premises
+
+http://it-trends.solarwinds.com/
+
+The story gets darker, 35% have migrated applications and infrastructure later to be brought back to on premise solutions according to a 2017 report from SolarWinds. That's a pretty bad failure rate. Not only that but the top reasoning for migrating back or never moving across was security and compliance, poor performance, cost, and technical challenges.
+
+## Security and compliance is not a technology issue
+
+The situation would be much improved if organisations had the skills necessary. Security and compliance is often blocked by a lack of knowledge regarding how to apply existing regulation to new technology.
+
+Cloud providers are working hard to provide far greater security than that of on-premise. Public cloud is making great progress with [Capital One](https://aws.amazon.com/solutions/case-studies/capital-one/) and [Pacific Life](https://aws.amazon.com/solutions/case-studies/pacific-life-insurance/) being a big case studies for AWS. Azure and Google are also building their own case studies in this space that has up till now been wary of public cloud.
+
+Researchers for Deutsche Bank are [calling 2017 the year of cloud migration](https://blogs.wsj.com/cio/2016/06/09/big-banks-starting-to-embrace-public-cloud-deutsche-bank-says/) for financial institutions. If the use cases are supported by the technology now, it's surely more a battle of politics and resourcing for highly regulated industries.
+
+## Cloud performance is achieved with new techniques
+
+An issue we see time and time again is the fitting of a square peg into a round hole. Whilst certainly tempting to try and run your applications within a cloud environment this will certainly lead to lots of head scratching.
```

Comment:
Repeated certainly - could tweak the line to read "Whilst it's tempting to try..."